pkg/limits/rate_limiting.go

```diff
@@ -77,6 +101,60 @@ var configs = []counterConfig{
 		Limit:  2000,
 		Period: 1 * time.Hour,
 	},
+	// JobThumbnail
+	{
+		Prefix: "job-thumbnail",
+		Limit:  5000,
+		Period: 1 * time.Hour,
+	},
+	// JobShareTrack
+	{
+		Prefix: "job-share-track",
+		Limit:  5000,
+		Period: 1 * time.Hour,
+	},
+	// JobShareReplicate
+	{
+		Prefix: "job-share-replicate",
+		Limit:  5000,
+		Period: 1 * time.Hour,
+	},
+	// JobShareUpload
+	{
+		Prefix: "job-share-upload",
+		Limit:  5000,
+		Period: 1 * time.Hour,
+	},
+	// JobKonnector
+	{
+		Prefix: "job-konnector",
+		Limit:  1000,
+		Period: 1 * time.Hour,
+	},
+	// JobZip
+	{
+		Prefix: "job-zip",
+		Limit:  1000,
+		Period: 1 * time.Hour,
+	},
+	// JobSendMail
+	{
+		Prefix: "job-sendmail",
+		Limit:  500,
+		Period: 1 * time.Hour,
+	},
+	// JobService
+	{
+		Prefix: "job-service",
+		Limit:  500,
+		Period: 1 * time.Hour,
+	},
+	// JobNotification
+	{
+		Prefix: "job-push",
+		Limit:  1000,
+		Period: 1 * time.Hour,
+	},
```

Comment:
Let's be more strict on the limits:

- share-replicate & share-upload: 500
- konnector & service: 100
- zip: 100
- sendmail & push: 30